roles/analysis/README.md

```diff
@@ -1,27 +1,65 @@
+Metadata is required for leap preupgrade analysis to run.  When a system uses CDN, this is provided automatically.  For systems not on CDN, package metadata files needs to be provided by URL in leapp_metadata_url. Once obtained during IPU, file is unpacked on IPU host under `/etc/leapp/files` directory.
+
+In an upcomoing release, the metadata will be included as part of the RPM install of leapp RPMs.
+
+For more information, refer to the knowledge article [Leapp utility metadata in-place upgrades of RHEL for disconnected upgrades (including Satellite)](https://access.redhat.com/articles/3664871).
+
+## Commonly modified variables
+
+| Name                  | Type | Default value           | Description                                     |
+|-----------------------|------|-------------------------|-------------------------------------------------|
+| leapp_upgrade_type    | String  | satellite | satellite, cdn or rhui |
+| leapp_preupg_opt | String | --no-rhsm when leapp_upgrade_type is connected | Upstream repository usage - whether to use RHSM |
+| leapp_metadata_url      | String |  | URL to the leapp metadata, usually over https |
+| leapp_enable_repos_args | String |  | --enablerepo (leapp_repos_enabled) or blank |
 
-## Role variables
 
-| Name                    | Default value         | Description                                         |
-|-------------------------|-----------------------|-----------------------------------------------------|
-| leapp_upgrade_type      | "satellite"           | Set to "cdn" for hosts registered with Red Hat CDN and "rhui" for hosts using rhui repos. |
-| leapp_metadata_url      |                       | See Requirements section above.                     |
-| leapp_answerfile        |                       | Optional multi-line string. If defined, this will be used as the contents of `/var/log/leapp/answerfile`. |
-| leapp_preupg_opts       |                       | Optional string to define command line options to be passed to the `leapp` command when running the pre-upgrade. |
-| post_reboot_delay       | 120                   | Optional integer to pass to the reboot post_reboot_delay option. |
+## Variables for Satellite based upgrades
+| Name                  | Type | Default value           | Description                                     |
+|-----------------------|------|-------------------------|-------------------------------------------------|
+| satellite_organization  | String | Example | Organization used in Satellite definition |
+| satellite_activation_key_leapp | String |  | Key used to identify activation key |
+| satellite_activation_key_pre_leapp | String |  | initial state of subscriptions and svc level |
+| satellite_activation_key_leapp     | String |  | Post-IPU state of subscriptions and svc level |
```

Comment:
I have no idea what this means. If role users really need to use these variables, we need to explain them much better than a short description that will fit in this table. Also, s/IPU/upgrade/g.